src/traffic_server/InkAPITest.cc

```diff
@@ -9002,9 +9002,9 @@ REGRESSION_TEST(SDK_API_DEBUG_NAME_LOOKUPS)(RegressionTest *test, int /* atype A
   }
 
   str = TSHttpEventNameLookup(TS_EVENT_IMMEDIATE);
-  if ((strlen(str) != strlen(event_name) || strcmp(str, event_name))) {
-    SDK_RPRINT(test, "TSHttpEventNameLookup", "TestCase1", TC_FAIL, "Failed on %d, expected %s, got %s", TS_EVENT_IMMEDIATE,
-               hook_name, str);
+  if (strstr(str, event_name) == nullptr) {
```

Comment:
Is this functionally equivalent? The original code is an anchored search, and this is not.

Reply:
I am changing the be behavior of HttpDebugNames::get_event_name() so that it outputs all the (slash-separated) synonyms for a particular event, rather than one selected on.  I changed this to just check if the synonym for the TS name for the tested event is included in the output string.